src/components/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -183,4 +171,26 @@ export class Breadcrumbs implements OnChanges {
         }
     }
 
+    private executeIEandEdgeEllipsisWorkAround(): void {
+        if (this.userAgent.isIE11) {
+            const newRouterLinks = this.routerLinks;
+            this.routerLinks = [];
+
+            setTimeout(() => {
+                this.routerLinks = newRouterLinks;
+            });
+        }
+
+        if (this.userAgent.isEdge) {
+            const newRouterLinks = this.routerLinks;
+
+            setTimeout(() => {
+                this.routerLinks = [];
+
+                setTimeout(() => {
+                    this.routerLinks = newRouterLinks;
```

Comment:
More setTimeouts please 😛

Reply:
Unfortunately that's the only way we found for making it work in IE and Edge (otherwise the ellipsis at the end of the collapsed breadcrumbs bar would not appear until moving the mouse over the breadcrumbs). If you have another idea, we would appreciate it :)

Reply:
No, I believe you - sounds like much real-world trial-and-error was taken before arriving at this fix! Is it not something that can be solved by injecting the ChangeDetectorRef and calling markForCheck()?

Reply:
We've tried various things and just to make sure, we've tried the ChangeDetectorRef solution again, (calling first markForCheck() and then detectChanges() to force a detection cycle) but it didn't work.